pysages/ssages/methods/umbrella.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Is this based on another implementation? If not I've been trying to have anything that is not derived from implementations in SSAGES MIT licensed.

Reply:
No, it's not based on anything.
I did pay much attention to the license.
I am fine with any